notebooks/hello_world_thunderfx.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>"to accelerate [a] PyTorch program"</p><p> </p><p>"... and evaluate its performance in both inference (forward-only) and training (forward + backward)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>